command/meta.go

```diff
@@ -67,7 +71,15 @@ func (m *Meta) Client() (*api.Client, error) {
 	if m.ForceAddress != "" {
 		config.Address = m.ForceAddress
 	}
-
+	if v := os.Getenv(EnvVaultCACert); v != "" {
+		m.flagCACert = v
+	}
+	if v := os.Getenv(EnvVaultCAPath); v != "" {
+		m.flagCAPath = v
+	}
+	if v := os.Getenv(EnvVaultInsecure); strings.ToLower(v) == "true" || strings.ToLower(v) == "yes" {
```

Comment:
Just a style thing. ParseBool has a set number of true/false values with other values throwing an error. This allows only very specific (read: longform) true values with anything else being ignored.

If you'd prefer ParseBool behavior I can change it.